etl.py

```diff
@@ -7,17 +7,22 @@
 from utils.gc_notify import gc_notify_log
 
 
-def setup_logging(log_filename):
+def setup_logging(log_folder, log_filename):
     """
     Set up logging to redirect stdout and stderr to log files.
 
     Args:
+        log_folder (str): The folder where the log file should be created.
         log_filename (str): The name of the log file.
 
     Returns:
         None
     """
-    logging.basicConfig(filename=log_filename, level=logging.INFO)
+    if not os.path.exists(log_folder):
+        os.makedirs(log_folder)
```

Comment:
@emilymendelson Can you do this? When you are writing the processed data in ltsa_parser? It basically creates the folder, if it doesn't exist. We don't have to jump through hoops anymore as long as that path being passed is within `./data/` as it has the appropriate permissions. Check the Dockerfile.